ui/admin/app/templates/scopes/scope/policies/policy/index.hbs

```diff
@@ -9,10 +9,11 @@
   </page.breadcrumbs>
 
   <page.header>
-    <Hds::Text::Display @tag='h2'>
+    <h2>
```

Comment:
the reason behind not using this new component is due to the lack of margin/padding and I did not want to add it individually, this should be handled in a common place since we will be using this everywhere and will need to test those before adding margins. I'll add an HDS migration ticket for this
<img width="1129" alt="Screenshot 2024-01-19 at 12 34 16 PM" src="https://github.com/hashicorp/boundary-ui/assets/15043878/0c96ac4a-b607-4aeb-91c9-5eb3c1eebf10">